solutions/ocp-ai/variables.tf

```diff
@@ -0,0 +1,53 @@
+# tflint-ignore: all
+variable "default_worker_pool_machine_type" {
+  type        = string
+  description = "The machine type for worker nodes."
+  default     = "bx2.8x32"
```

Comment:
Ensure this default value supports AI workload

Reply:
In that case, GPU machine type should be provided. I am checking this.

Reply:
This supports AI workloads and also additional worker pool will have GPU machines.